components/AddDonation.js

```diff
@@ -0,0 +1,136 @@
+import styled from "styled-components";
+import { useState } from "react";
+import Counter from "./Counter";
+
+export default function AddDonation({ createDonation }) {
+  const [toggleAddButton, setToggleAddButton] = useState(true);
```

Comment:
You can switch the meaning of this boolean, that it is true if the form is visible and call it isFormVisible

Reply:
great suggestion! changed it to isFormVisible!